zeitschrift-fur-medienwissenschaft.csl

```diff
@@ -20,493 +14,297 @@
     <category field="humanities"/>
     <issn>1869-1722</issn>
     <summary>Style for zfm. Zeitschrift für Medienwissenschaft. Citations in notes; no bibliography.</summary>
-    <updated>2012-09-27T22:06:38+00:00</updated>
+    <updated>2014-09-13T22:01:31+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
-  <!--
-================================================================================
-   TODO
-================================================================================
--->
-  <!--
-================================================================================
-   TERMS
-================================================================================
--->
+  <!-- HINWEISE
+       ========
+       - Bei Zeitschriftenartikeln (article-journal, article-magazine) muss
+         der Verlagsort (publisher-place) ins Zotero-Feld "Extra" eingetragen
+         werden, und zwar nach dem Muster "{:publisher-place: Drexel Hill,
+         Pennsylvania}".
+       - Bei Filmen (motion_picture) muss das Land ins Zotero-Feld "Extra"
+	 eingetragen werden, und zwar dem Muster "{:publisher-place: USA}".
+       - Das Originalpublikationsjahr muss ins Zotero-Feld "Extra" eingetragen
+         werden, und zwar nach dem Muster "{:original-date: 1964}".
+       - Bei Reihen (series) muss die Abkürzung "Bd. " als Präfix für die
+	 "Nummer der Reihe" ins Zotero-Feld "Nummer" eingetragen werden (nach
+	 dem Muster "Bd. 7").
```

Comment:
I'm having second thoughts here: The problem is that numbers should be
displayed with different prefixes (or no prefix at all) for different
collections/series (e.g. "stw 651", "Literatur- und Medienanalyen, Bd.
3", "Schriftenreihe Schweizerische Postgeschichte, Nr. 2"). I see no
other solution than for the prefix to be added manually by the user to
the field collection-number/Series number. One could then -- at least --
use Philipp's construct to distinguish the cases where a comma should be
added (field is not numeric) or not (field is numeric). Suggestions?

> On 2015-01-04T17:04:14+0100, Philipp Zumstein said:
> -   TERMS

> -================================================================================
> --->
> +  <!-- HINWEISE
> +       ========
> +       - Bei Zeitschriftenartikeln (article-journal, article-magazine) muss
> +         der Verlagsort (publisher-place) ins Zotero-Feld "Extra" eingetragen
> +         werden, und zwar nach dem Muster "{:publisher-place: Drexel Hill,
> +         Pennsylvania}".
> +       - Bei Filmen (motion_picture) muss das Land ins Zotero-Feld "Extra"
> +	 eingetragen werden, und zwar dem Muster "{:publisher-place: USA}".
> +       - Das Originalpublikationsjahr muss ins Zotero-Feld "Extra" eingetragen
> +         werden, und zwar nach dem Muster "{:original-date: 1964}".
> +       - Bei Reihen (series) muss die Abkürzung "Bd. " als Präfix für die
> +	 "Nummer der Reihe" ins Zotero-Feld "Nummer" eingetragen werden (nach
> +	 dem Muster "Bd. 7").

> Wie wäre es mit einem folgendem Konstrukt: ``` <choose> <if
> is-numeric="volume"> <group delimiter=" "> <number
> variable="volume"/> <text term="volume" form="short"/> </group>
> </if> <else> <text variable="volume"/> </else> </choose> ```
> Standardmässig wird das Feld nur eine Zahl sein und dann mit "Bd."
> angehängt. Falls der Benutzer aber explizit eine andere Bezeichnung
> möchte, kann er dies im gleichen Feld mitangeben und diese wird dann
> so abgedruckt wie eingegeben.

> --- Reply to this email directly or view it on GitHub:
> https://github.com/citation-style-language/styles/pull/1313/files#r22440251

--
Till A. Heilmann, Dr. phil.

Online-Texte http://www.tillheilmann.info/german.php#OnlineTexts

Universität Siegen
Philosophische Fakultät
Medienwissenschaftliches Seminar
Adolf-Reichwein-Straße 2, Raum AR-K 611
57076 Siegen
Telefon: +49 271 740-4791
Telefax: +49 271 740-2731
www.uni-siegen.de/phil/medienwissenschaft/
www.multimediale-systeme.de